runtime/tests/interpreter/attachments_test.go

```diff
@@ -1989,107 +1988,6 @@ func TestInterpretAttachmentSelfAccessMembers(t *testing.T) {
 	require.NoError(t, err)
 }
 
-func TestInterpretAttachmentMappedMembers(t *testing.T) {
```

Comment:
This run-time test is failing, which I'm not sure why. There is a corresponding checker test, so this is mostly a second layer of defence.

Reply:
These previously failed at runtime with an invalid transfer error, which I think will no longer occur because the value being transferred is now being converted at runtime with these changes. So this removal makes sense to me.